### DISCUSSION TITLE
{   "errors": [     {       "message": "You don't have permission to access this.",       "extensions": {         "code": "FORBIDDEN"       }     }   ] }

### DISCUSSION THREAD

**Discussion by thesocialdeck** [directus/directus#4731] [Archive]
Hi all,

Trying to get up and running with a SQLite database on directus open source. I connected to an existing database and when I run directus and try to access a table i get the following error

```json
{
  "errors": [
    {
      "message": "You don't have permission to access this.",
      "extensions": {
        "code": "FORBIDDEN"
      }
    }
  ]
}
```

Any advice would be much appreciated.

Steve

**Comment by rijkvanzanten** [comment]
It's most likely one of:

1) You don't have permissions to access that table from the current role

Make sure you have the correct permissions configured for your current user role, and or the public role if you're not authenticating

2) You're using the wrong endpoint

The same forbidden error is thrown for 404s, so external actors can't scrape your data model 👍🏻

**Reply by QYongHao** [reply]
I'm trying to create a new collection named **articles** but I get the same 403 error even while using the admin account.

```
import { Directus } from "@directus/sdk";

const directus = new Directus("http://localhost:8055");

directus.auth.login({
  email: "admin@example.com",
  password: "d1r3ctu5",
}).then(() => {
  return directus.collections.createOne({ collection: 'articles' });
}).then(console.log)
  .catch(console.log);
```
The error I'm getting is:
```
  errors: [
    {
      message: "You don't have permission to access this.",
      extensions: { code: 'FORBIDDEN' }
    }
  ]
```

I am able to read the collections, I just can't create any. Do you have any advice on what might be the issue?

**Reply by QYongHao** [reply]
Just figured it out. needed to add a meta object like so:
```
{ collection: 'articles', meta: {
      "collection": "articles",
      "icon": "article",
      "note": "Blog posts",...
```

**Comment by Heartnett** [comment]
@rijkvanzanten I am facing the same issue.

The user account I am using has full admin access, which is trying to create an item in a collection.

Is there some way to enable debugging to determine what it is that is causing the problem, or maybe view a log or log file?

I'm getting this error (docker version) both with the SDK (websockets) and REST calls (postman).

**Reply by rijkvanzanten** [reply]
Can you check the collection and field names are correct (they're case sensitive)?

**Reply by Heartnett** [reply]
Yes, I am aware that all fields are case sensitive. I have checked several times.

**Reply by Heartnett** [reply]
The problem was related to a flow. Its solved now. Still I think there should be a way to enable detailed errors - environment variable flag accompanied by warning messages/banners in directus or something.

**Reply by br41nslug** [reply]
Then please do follow/upvote the feature request for it 😄 https://github.com/directus/directus/discussions/4368
